app/controllers/timecards_controller.rb

```diff
@@ -1,6 +1,6 @@
 class TimecardsController < ApplicationController
   before_filter :authenticate_user!
-  load_and_authorize_resource
+  load_and_authorize_resource except: :verify
```

Comment:
Rather than carving out an exception for verify, what if we added verify to the Ability class here ?
https://github.com/ReadyResponder/ReadyResponder/blob/development/app/models/ability.rb#L35
If you look at line 27 of that file, you can see how custom actions were added to the Person class.
See further comments in def verify